integration/nevermined/NFT721.test.ts

```diff
@@ -41,7 +41,7 @@ describe('Nfts721 operations', async () => {
                 getMetadata(),
                 artist,
                 new AssetRewards(),
-                nft.options.address
+                nft.options.address,
```

Comment:
I guess need to run eslint to remove this warning...

Reply:
:+1: